smokey/src/main/java/frc/robot/control/SubsystemCollection.java

```diff
@@ -24,12 +24,16 @@ public class SubsystemCollection
     private EveryBotPickerSubsystem everyBotPickerSubsystem = null;
     private PowerDistributionPanelWatcherSubsystem powerDistributionPanelWatcherSubsystem = null; 
     private StabilizerSubsystem stabilizerSubsystem = null;
+    private CameraSubsystem cameraSubsystem = null;
 
     /**
      * Default constructor
      */
     public SubsystemCollection() {}
 
+    public CameraSubsystem getCameraSubsystem() { return cameraSubsystem; }
```

Comment:
This class doesn't follow aphetically method orientation as getter/setters are grouped.  Likely we should change that in season.  Just an FYI that we should be more rigorous.